msi/make.cmd

```diff
@@ -0,0 +1,33 @@
+rmdir /s/q build
+
+rem TODO: Where should the git-credential-winstore, gcm and posh-git binaries
+rem       be found/stored?
+
+rem TODO: Generate GitComponents.wxs from ..\make-file-list.sh
+rem
+rem       1. Files installed to `bin\` folder use Directory='BinFolder' and
+rem          all others use the pattern Directory='INSTALLFOLDER:\path\'.
+rem
+rem       2. The `git.exe` file in `BinFolder` is given the explicit
+rem          Id='GetExe' all other files are anonymous (do not have an Id).
+
+rem Build the .msi.
+rem
+rem TODO: Correctly set the bind paths (change the -b gitfileshere switch)
+rem       to the real source location for the binaries to include in the
+rem       installation package.
+rem
+rem TODO: Set the output paths appropriately.
```

Comment:
No, I simply meant for the output of `candle.exe` to flow into the input of `light.exe` and `light.exe` to put the output where you wanted it.